pkg/storegateway/partitioner.go

```diff
@@ -13,6 +13,18 @@ import (
 	"github.com/prometheus/client_golang/prometheus/promauto"
 )
 
+type blockPartitioners struct {
+	Chunks, Series, Postings Partitioner
```

Comment:
[nit] No need to expose them.

```suggestion
	chunks, series, postings Partitioner
```